public/app/core/components/sidemenu/SideMenuItem.tsx

```diff
@@ -41,7 +37,7 @@ const SideMenuItem = ({
       <span className="icon-circle sidemenu-icon">{children}</span>
     </Link>
   ) : (
-    <button className={cx(resetButtonStyles, 'sidemenu-link')} onClick={onClick} aria-label={label}>
+    <button className="sidemenu-link" onClick={onClick} aria-label={label}>
```

Comment:
Maybe a good moment to refactor this classname (and other used in this pr) to Emotion? 🤔

Reply:
great minds! i'm currently working on a follow up PR which does just that 🥳